INSTALL.rst

```diff
@@ -19,52 +19,25 @@ Installing
 Installing an official release
 ==============================
 
-Matplotlib and most of its dependencies are all available as wheel packages for
-macOS, Windows and Linux distributions::
+Matplotlib and most of its dependencies are all available as wheel
+packages for macOS, Windows and Linux distributions::
 
-  pip install -U matplotlib
+  python -m pip install -U pip
+  python -m pip install -U matplotlib
 
+.. note::
 
-Third-party distributions of Matplotlib
-=======================================
-
-Scientific Python distributions: Conda, Canopy...
--------------------------------------------------
-
-The first option is to use one of the pre-packaged Python distributions that
-already provide Matplotlib built-in.  Both `Anaconda
-<https://www.continuum.io/downloads/>`_ and `Canopy
-<https://www.enthought.com/products/canopy/>`_ are both excellent choices that
-"just work" out of the box for Windows, macOS and common Linux platforms.  Both
-of these distributions include Matplotlib and *lots* of other useful tools.
-
-Linux : using your package manager
-----------------------------------
-
-If you are on Linux, you might prefer to use your package manager.  Matplotlib
-is packaged for almost every major Linux distribution.
+   The following backends work out of the box: Agg, TkAgg, ps, pdf and svg.
```

Comment:
Yes (although tkagg may fail if your distro packages python-tk separately and you don't have it installed).